apps/service/mailer/mailer.go

```diff
@@ -30,15 +31,20 @@ func CreateClient() (*ses.SES, error) {
 type SendOptions struct {
 	From    string
 	ReplyTo string
-	To      string
+	To      []string
 	Subject string
 	Body    string
 }
 
 func Send(client *ses.SES, options SendOptions) error {
+	toAddresses := make([]*string, len(options.To))
+	for i, address := range options.To {
+		toAddresses[i] = aws.String(address)
+	}
+
 	emailInput := &ses.SendEmailInput{
 		Destination: &ses.Destination{
-			ToAddresses: []*string{aws.String(options.To)},
+			ToAddresses: toAddresses,
```

Comment:
oh at least we are juts sending 1 email w/ all of them on it instead of 1 email per recipient... i suppose i feel quite a bit better about it w/ that in mind.